bpf/Makefile.bpf

```diff
@@ -27,8 +27,8 @@ HOST_CC    ?= gcc
 HOST_STRIP ?= strip
 
 ifeq ($(IMAGE_CROSS_TARGET_PLATFORM),linux/arm64)
-  HOST_CC ?= aarch64-linux-gnu-gcc
```

Comment:
I suggested `?=` here as I was thinking one should be able to override these, however since the whole condition is based on a very explicit `IMAGE_CROSS_TARGET_PLATFORM` parameter, I think it's perfectly fine to use `=` or even `:=` here.

Reply:
The thing is that `?=` will not take any effect, because after going to this if statement, both variables are already set in lines 26-27.

But I have an another idea. If we really want to keep the ability of overriding these in any case, I think something like that should work:

```
ifeq ($(IMAGE_CROSS_TARGET_PLATFORM),linux/arm64)
  HOST_CC ?= aarch64-linux-gnu-gcc
  HOST_STRIP ?= aarch64-linux-gnu-strip
else
  HOST_CC ?= gcc
  HOST_STRIP ?= strip
endif
```

Or another idea - get rid of this if statement and just set those two variables explicitly in Dockerfile.

WDYT?

Reply:
AFAICT, @errordeveloper wasn't necessarily advocating for any particular change here. The current version seems fine to me. If we change our minds, this is always something we can iterate on further but for now I think that this is good to merge.

Reply:
:+1: